### ISSUE TITLE
Sync: Clan Properties

### ISSUE THREAD

**Issue by garrettluskey** [araex/BannerlordCoop#852]
## Description

1. Sync the values in the **Synced** list below.
2. Create an end to end (E2E) tests for all properties (1 for server changing the value and 1 for client changing the value if applicable)
3. Create a command to test changing each property in game [see similar command](https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/development/source/GameInterface/Services/Towns/Commands/TownDebugCommand.cs#L455)

Also see additional information at the bottom for more information and templates

**Key**
Server side only (only allow the running of the function on the server side)
Client side only (only allow the running of the function on the client side)
Client side sync (request the server to change the value, server then changes the value and replies to the client allowing the change)
Server side sync (only server allows running of the function and send to all clients the value changed and clients update the value/call original function)

**Synced**

| Method/Field Name | Sync Type | Additional Information |
|:-----------------|:---------:|:--------------|
| Name | Server side sync | |
| InformalName | Server side sync | |
| Culture | Server side sync | |
| LastFactionChangeTime | Server side sync | |
| AutoRecruitmentExpenses | Server side sync | |
| IsNoble | Server side sync | |
| TotalStrength | Server side sync | This might have a lot of messages |
| MercenaryAwardMultiplier | Server side sync | |
| LabelColor | Server side sync | |
| InitialPosition | Server side sync |
| IsRebelClan | Server side sync | |
| IsUnderMercenaryService | Server side sync | |
| Color | Server side sync | |
| Color2 | Server side sync | |
| BannerBackgroundColorPrimary | Server side sync | |
| BannerBackgroundColorSecondary | Server side sync | |
| BannerIconColor | Server side sync | |
| _midPointCalculated | Server side sync | This might have a lot of messages |
| Renown | Server side sync | |
| NotAttackableByPlayerUntilTime | Server side sync | Might be used to prevent ai from attacking other players |

**Deferred**

| Method/Field Name | Remarks |
|:-----------------|:---------:|
| Kingdom | Synced by _kingdom |
| Influence | Synced by _influence |
| BasicTroop | Synced by _basicTroop |
| Banner | Synced by _banner |
| Tier | Synced by _tier |
| Aggressiveness | Synced by _aggressiveness |
| TributeWallet | Synced by _tributeWallet |
| HomeSettlement | Synced by _home |
| DebtToKingdom | Synced by _clanDebtToKingdom |

**Externally Deferred**

None

**Non-Synced**

| Method/Field Name | Remarks |
|:-----------------|:---------:|
| EncyclopediaText | Only set on load |
| IsMinorFaction | Only set on load |
| IsOutlaw | Only set on load |
| IsNomad | Only set on load |
| IsMafia | Only set on load |
| IsClanTypeMercenary | Only set on load |
| IsSect | Only set on load |
| AlternativeColor | Only set on load |
| AlternativeColor2 | Only set on load |
| IsBanditFaction | Only set on load |
| MainHeroCrimeRating | Not used |

## Intended Design

You are expected to only sync the property setters.
Try using the new [autosync tool](https://github.com/Bannerlord-Coop-Team/BannerlordCoop/wiki/Property-AutoSync)

## Location

Create a branch based from [development](https://github.com/Bannerlord-Coop-Team/BannerlordCoop)

## Related Issues

Blocked by:
- #850

## Requirements

N/A

## Additional information

Using the [GameInterface Service Tempale](https://github.com/Bannerlord-Coop-Team/BannerlordCoop/tree/development/source/GameInterface/Services/Template) create commands to test the synced values.

## Definition of Done
- [ ] Class level comments exist for all new classes.
- [ ] XUnit E2E tests exist for every property on the server (and client if applicable)
- [ ] Each property has a command getter and setter
